catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -2482,73 +2418,45 @@ protected ResourceResponse getResource(ResourceRequest resourceRequest, boolean
                 resourceSourceName = resolvedSourceId;
             }
 
-            String key;
-            try {
-                key = new CacheKey(metacard, resourceRequest).generateKey();
```

Comment:
Probably, although I'd want to check with @lessarderic since I think he might have mentioned that we won't need CacheKey soon, though it's possible I heard that wrong